.github/workflows/readme.yml

```diff
@@ -5,8 +5,6 @@ on:
   push:
     branches:
       - master
-    paths:
-      - '_examples/**' 
```

Comment:
removing `paths` condition will lead to infinite github-action trigger.

Reply:
not actually, the path is used to trigger the workflow when files has been updated under the mentioned `paths`. so after removing this code pipeline will be triggered PR is merged or code is directly pushed to master branch.

Reply:
Did we got errors anything related to `paths`? it was not before branch protection issue fix.

Reply:
- Readme action gets triggered when changes in master branch are made.
- The **Readme action** pushes code to `master` branch's Readme.md file(which is in root directory) and this `push` event will again trigger the **Readme action** which will push the changes in Readme.md and this will lead to infinite github-action trigger.
- To prevent this, we mentioned `paths:` condition so that workflow will trigger when changes happen in that path only and `Readme.md` is not present in that path `'_examples/**' ` which will prevent infinite github-action trigger.

Reply:
Now `paths-ignore` argument is used, so that workflow can be triggered when any other files has changed excluding readme.